docs/guides/4-custom-rulesets.md

```diff
@@ -402,38 +402,51 @@ Rulesets can then reference aliases in the [given](#given) keyword, either in fu
 
 Previously Spectral supported exceptions, which were limited in their ability to target particular rules on specific files or parts of files, or changing parts of a rule. Overrides is the much more powerful version of exceptions, with the ability to customize ruleset usage for different files and projects without having to duplicate any rules.
 
-Overrides can be used to:
+Overrides can be used to apply rulesets on:
 
-- Override rulesets to apply on particular files/folders `files: ['schemas/**/*.draft7.json']`
-- Override rulesets to apply on particular JSONPath's `files: ['**#/components/schemas/Item']`
-- Override rulesets to apply on particular formats `formats: [jsonSchemaDraft7]`
+- Particular formats `formats: [jsonSchemaDraft7]`
```

Comment:
I moved formats above files/folders so that the options would be ordered from "general" -> "specific".